cmd/config/aws/aws.go

```diff
@@ -0,0 +1,34 @@
+package aws
+
+import (
+	"github.com/armory/armory-cli/pkg/cmdUtils"
+	cliconfig "github.com/armory/armory-cli/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+const (
+	configAWSShort = "Manage CD-as-a-Service AWS Access"
+	configAWSLong  = "Manage CD-as-a-Service AWS Access\n\n" +
+		"For usage documentation, visit TODO"
+	configAWSExample = "armory aws <subcommand>"
+)
+
+func NewAWSCmd(configuration *cliconfig.Configuration) *cobra.Command {
+	command := &cobra.Command{
```

Comment:
maybe keep the command hidden for now, especially if there's a TODO in the help fields?